domains/kietdo.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "kietdo-dev",
+    "email": "lionvs12345@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://kietdo-portolio.vercel.app"
```

```suggestion
    "CNAME": "kietdo-portolio.vercel.app"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).